### DISCUSSION TITLE
Android progress bar on open paypall

### DISCUSSION THREAD

**Discussion by YankovWeb** [Adyen/adyen-react-native#317] [Q&A] [answered]
How to remove the android progress bar on opening paypall?

**Comment by descorp** [comment, marked as answer]
Hey @YankovWeb 

You can always override [pending_payment](https://github.com/Adyen/adyen-react-native/blob/develop/android/src/main/res/layout/pending_payment.xml) layout locally.

However, if the shopper decides to open PayPal (or any other redirect flow payment method) in external browser - "in-app" browser will disappear and there is no way to tell if shopper cancel the payment or still in the process. This could lead to poor UX and even overcharges.
